src/components/pages/weather/elements/WeatherTempCell.js

```diff
@@ -2,12 +2,27 @@ import React, { Component } from 'react';
 
 export default class WeatherTempCell extends Component {
   render() {
-    const averageTemp = this.props.dayTemperatureData.avg;
-    const maxTemp = this.props.dayTemperatureData.max;
-    const minTemp = this.props.dayTemperatureData.min;
+    const averageTemp = this.props.dayTemperatureData.avg.toFixed(3);
+    const maxTemp = this.props.dayTemperatureData.max.toFixed(3);
+    const minTemp = this.props.dayTemperatureData.min.toFixed(3);
+    let temp = 'boiling';
+    if (averageTemp >= -20) {
+      temp = 'hot';
+    } else if (averageTemp >= -40) {
+      temp = 'warm';
+    } else if (averageTemp >= -60) {
+      temp = 'normal';
+    } else if (averageTemp >= -80) {
+      temp = 'cool';
+    } else if (averageTemp >= -100) {
+      temp = 'cold';
+    } else {
+      temp = 'freezing';
+    }
+    const className = `temperature-cell-${temp}`;
```

Comment:
Great idea! I would extract this into a separate function so that in the `render` method we can call:

`const temperatureCellClass = getTemperatureCellClass(averageTemp)`

It's often a good idea if your functions are getting long or bloated to see if there are any bits that could be factored out into another function.